scripts/check_backend_parity.py

```
@@ -727,7 +773,22 @@ def carrying(wrappers, names):
                     carrying(nat_wrappers, set(nat_wrappers) - set(mod_wrappers)),
                     WRAPPERS_BY_DESIGN, WRAPPERS_KNOWN_GAPS)
 
-    if (v1 or v2 or v3 or v4 or v5 or s1 or s2 or s3 or s4 or s5):
+    # ...and a payload that runs one of those wrappers by bare NAME, which
+    # leaves the choice of file to whichever PATH the caller happens to have.
+    # Reported as module-side: the payload is shared, so the call is a hazard
+    # for whichever backend's PATH does not carry the CLI, and naming a side
+    # would be arbitrary.
+    bare = bare_cli_calls(set(mod_wrappers) & set(nat_wrappers))
+    print(f"\nshared payloads running a generated CLI by bare name "
```

Ruff (0.16.3)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)